### ISSUE TITLE
Globe planétaire lévitant

### ISSUE THREAD

**Issue by petrachi** [liotag/impression-3d#6]
Le but serait d'imprimer un globe terrestre qui léviterai en l'air grâce à un système d'aimant.

L'idée est de suivre ce tutoriel https://all3dp.com/1/weekend-project-3d-printed-levitating-plant/ en remplaçant la plante par un globe terrestre comme ici : https://www.myminifactory.com/object/3d-print-icosahedron-earth-folding-polyhedra-81866

Ces deux vidéos expliquent comment réaliser le globe terrestre soi même : 
https://www.youtube.com/watch?v=KV2AjyowXX4 et https://www.youtube.com/watch?v=DXeR6tmcZCE

On peut aussi imaginer des variantes avec des modèles de la lune, ou de satelittes artificiels comme on peut en trouver sur le site de la nasa

**Comment by liotag** [comment]
t'es sûr du premier lien YT? ça montre comment imprimer une coque de smartphone  😮 
la seconde est folle n’empêche, pour l'assemblage du polyhedre sans colle
chouette projet!

**Comment by petrachi** [comment]
Oui, ça explique la technique pour utiliser plusieurs couleurs sur les premiers layers de l'impression. Technique utilisée dans la seconde vidéo pour dessiner les reliefs du globe
